indico/modules/designer/util.py

```diff
@@ -38,3 +40,11 @@ def get_all_templates(obj):
 def get_inherited_templates(obj):
     """Get all templates inherited by a given event/category"""
     return get_all_templates(obj) - set(obj.designer_templates)
+
+
+def get_default_template_on_category(category):
+    if category.default_ticket_template:
+        return category.default_ticket_template
+    parent_chain = reversed(category.parent_chain_query.options(joinedload('default_ticket_template')).all())
```

Comment:
IMO it's more readable to do the `reversed()` below, i.e. `for category in reversed(parent_chain)`